common/client/client.go

```diff
@@ -0,0 +1,27 @@
+package client
```

Comment:
if we renamed this dir/pkg to cli, we could get rid of the import stutter (client.*Client -> cli.Client)

Reply:
There's another package named cli in the client package, which has a cli.go, so it would be the same issue if we renamed it.